content/tutorials/git_authentication/index.Rmd

```diff
@@ -0,0 +1,266 @@
+In earlier days, it was sometimes needed to store your PAT in a `.Renviron` text file which you store only locally (the file will in that case contain a line GITHUB_PAT=...).
+This is no longer needed and should be avoided because of the risk to expose the PAT when that file is accidentally put online.
+You can check if you have previously set a GITHUB_PAT in your `.Renviron` with the R command `usethis::edit_r_environ()`.
+
+## Creating Personal Access Tokens (PAT)
+
+We move the steps about creating a PAT upfront, because this includes guidelines to store the PAT to the Git Credential Manager so they can be discovered automatically.
+
+Personal access tokens can only be used for HTTPS Git operations.
+When you create a PAT, you will be asked which scopes should be allowed (i.e. what rights do you give).
+We suggest that you stick with the scopes recommended by the `usethis` package.
+Executing the following command takes your browser to a pre filled form with the recommended scopes (repo, user, workflow) where you can create your PAT.
+
+```{r eval=FALSE}
+usethis::create_github_token()
+```
+
+After you have done this, you can store this PAT in the Git Credential Manager using this code:
+
+```{r eval=FALSE}
+gitcreds::gitcreds_set()
+```
+
+which will open a prompt where you can paste your PAT (or replace an old one).
+
+If you want to know more about PATs read [this section from happy git with R](https://happygitwithr.com/https-pat.html#get-a-pat) and [this section from the usethis package](https://usethis.r-lib.org/articles/git-credentials.html#get-a-personal-access-token-pat).
+
+After you've done this, you can check your authentication settings.
+The `usethis` package has a function to get a situation report on your current Git/GitHub status, including information about authentication:
+
+```{r eval=FALSE}
+usethis::git_sitrep()
+```
+
+If you see warnings about git_vaccinated, it is recommended to execute
+
+```{r}
+usethis::git_vaccinate()
+```
+
+which adds a global `.gitignore` file that decreases the chance that you accidentally leak credentials.
+
+The situation report should normally report that the default Git protocol is `https`, which we will discuss next.
+
+## <a name="https"></a>Using the HTTPS protocol
+
+The HTTPS protocol is the GitHub default URL transport protocol that uses the Transport Layer Security (TLP) encryption protocol to encrypt communications.
+To learn more about what https is, [read this explanation](https://www.cloudflare.com/learning/ssl/what-is-https/).
+
+The usual way to choose the protocol is when you clone a repository to your computer.
+You can do this in various ways:
+
+- Manual way:
+    - Go to https://github.com/inbo/tutorials press clone button and copy the URL (default is HTTPS: https://github.com/inbo/tutorials.git)
+    - Open RStudio -> File -> New project -> Version control -> Git -> paste https://github.com/inbo/tutorials.git -> create project
```

Comment:
```suggestion
    - Open RStudio -> File -> New project -> Version control -> Git -> paste `https://github.com/inbo/tutorials.git` -> create project
```

Otherwise it all displays as hyperlinks: 

![afbeelding](https://user-images.githubusercontent.com/19164640/166716190-3be8a443-62e9-4cbb-a38c-5bc7aa87ffd8.png)